modules/hcp-ibmz-update-reg-ca.adoc

```diff
@@ -0,0 +1,65 @@
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-update-registry-ca-private-registry_{context}"]
+= Update the registry certificate authority in the AgentServiceConfig resource with the mirror registry
+
+When you use a mirror registry for images, agents need to trust the registry's certificate to securely pull images. Add the certificate authority of mirror registry to the `AgentServiceConfig` custom resource by creating a `ConfigMap`.
+
+.Prerequisites
+
+* You must have installed {mce}.
+
+.Procedure 
+
+. In the same namespace where you installed {mce-short}, create a config map with the mirror registry details. This config map ensures that you grant the hosted cluster workers the capability to retrieve images from the mirror registry.
```

Comment:
```suggestion
. In the same namespace where you installed {mce-short}, create a `ConfigMap` resource with the mirror registry details. This `ConfigMap` resource ensures that you grant the hosted cluster workers the capability to retrieve images from the mirror registry.
```

Reply:
I _think_ in these instances you are referring to a specific `ConfigMap` resource and not the general sense of config maps.